### ISSUE TITLE
Code for Causal TAE

### ISSUE THREAD

**Issue by anonymous-ai-model** [zju3dv/MotionStreamer#6]
Thank you for the great work! Could you please point me to the code for Causal TAE? I would really appreciate it.

**Comment by Li-xingXiao** [comment]
> oint me to the code for Causal TAE? I would really appreciate it.

Hi! Thank you for your attention. The code for Causal TAE is still under prepare and will be release soon.

Best Wishes!

**Comment by anonymous-ai-model** [comment]
@Li-xingXiao  Do you have some reference code from other repos just for TAE? Even if there is no causal that's fine. I need to see how to train simple TAE or VAE without vector quantization?

**Comment by Li-xingXiao** [comment]
> [@Li-xingXiao](https://github.com/Li-xingXiao) Do you have some reference code from other repos just for TAE? Even if there is no causal that's fine. I need to see how to train simple TAE or VAE without vector quantization?

Yes, I recommend you to have a look at [MagvitV2](https://github.com/lucidrains/magvit2-pytorch) and [MAR](https://github.com/LTH14/mar).

Best Wishes!

**Comment by anonymous-ai-model** [comment]
@Li-xingXiao I think there may have been a misunderstanding. The papers you mentioned involve quantization in VAEs, but I'm looking for a simpler VAE or TAE setup like the one in your paper. Specifically, I need a model similar to your TAE, but without the causal component—just the KL divergence and reconstruction loss.

If you have some suggestion for related papers, I really appreciate your help in this matter!

**Comment by Li-xingXiao** [comment]
> apers you mentioned involve quantization in VAEs, but I'm looking for a simpler VAE or TAE setup like the one in your paper. Specifically, I need a model similar to your TAE, but without the causal component—just the KL divergence and reconstruction loss.

Hi! Sorry for the misunderstanding. [Sigma-VAE](https://github.com/orybkin/sigma-vae-pytorch/tree/master) offers a simple VAE without the causal component and quantization. I recommend you to have a look at it. However, it is designed for image compression, and it does not include temporal convolution. If you need a simple VAE without quantization, you can take Sigma-VAE, but if you need a (causal) TAE without quantization, please wait for our code implementation soon.
